frontend/src/metabase/query_builder/actions/core/utils.ts

```diff
@@ -69,3 +70,15 @@ export function getAdHocQuestionWithVizSettings(options: {
 
   return question;
 }
+
+export function syncCardParametersWithTemplateTags(question: Question) {
```

Comment:
nit: I wonder whether this function should be part of `Question` class (since it receives a `Question` and returns a `Question`)

Reply:
Good idea, fixed in https://github.com/metabase/metabase/pull/50728/commits/a8e3ee751278a50a7d26cd0e5b275060f3a0e94e